### ISSUE TITLE
Looks like dependencies are outdated / requires documentation update

### ISSUE THREAD

**Issue by pschakravarthi** [RocketChat/Rocket.Chat.Electron#1942]
## My Setup

- Operating System: Mac Catalina
- App Version: 3.1.1
- Installation type: 

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description
$ yarn
yarn install v1.22.10
[1/5] 🔍 Validating package.json...
[2/5] 🔍 Resolving packages...
[3/5] 🚚 Fetching packages...
[4/5] 🔗 Linking dependencies...
warning " > @rocket.chat/eslint-config@0.4.0" has incorrect peer dependency "eslint@^4.19.1 || ^5.16.0".
warning "@rocket.chat/eslint-config > eslint-plugin-import@2.20.1" has incorrect peer dependency "eslint@2.x - 6.x".
[5/5] 🔨 Building fresh packages...
$ run-s install-app-deps clean
Watching /Users/chakri/mobile/react/2/2/Rocket.Chat.Electron-3.1.1 and all sub-directories not excluded by your .gitignore. Will not monitor dotfiles.
Found & ignored ./node_modules ; is listed in .gitignore

Starting: install-app-deps
node:internal/modules/cjs/loader:928
throw err;
^
Error: Cannot find module '/Users/chakri/mobile/react/Rocket.Chat.Electron/install-app-deps'
at Function.Module._resolveFilename (node:internal/modules/cjs/loader:925:15)
at Function.Module._load (node:internal/modules/cjs/loader:769:27)
at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
at node:internal/main/run_main_module:17:47 {
code: 'MODULE_NOT_FOUND',
requireStack: []
}
### Current Behavior
As per git repo README.md, I am not able to run yarn command successfully.

### Expected Behavior
Dependencies issues recorded in the documentation along with fix/ workaround

**Comment by nodeg** [comment]
I am not able to reproduce this on macOS Big Sur.

```
$ yarn
yarn install v1.22.10
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
[4/5] 🔗  Linking dependencies...
warning " > @rocket.chat/eslint-config@0.4.0" has incorrect peer dependency "eslint@^4.19.1 || ^5.16.0".
warning "@rocket.chat/eslint-config > eslint-plugin-import@2.20.1" has incorrect peer dependency "eslint@2.x - 6.x".
[5/5] 🔨  Building fresh packages...
$ run-s install-app-deps clean
yarn run v1.22.10
$ electron-builder install-app-deps
  • electron-builder  version=22.9.1
  • loaded configuration  file=/Users/dom/git/work/Rocket.Chat.Electron/electron-builder.json
✨  Done in 0.97s.
yarn run v1.22.10
$ rimraf app dist
✨  Done in 0.34s.
✨  Done in 38.64s.
yarn  41.81s user 28.92s system 182% cpu 38.835 total

$ yarn start                       
yarn run v1.22.10
$ run-s clean build:watch
$ rimraf app dist
$ rollup -c -w
rollup v2.34.0
bundles src/rootWindow.ts → app...
created app in 11.3s
bundles src/preload.ts → app...
created app in 8.1s
bundles src/injected.ts → app...
created app in 6.1s
bundles src/main.ts → app...
Starting main process...
created app in 6.4s

[2021-03-04 18:45:22] waiting for changes...
Checking for update
Update for version 3.2.0-snapshot is not available (latest version: 3.1.1, downgrade is disallowed).
```
The app starts as expected. 

I am not sure why you ran `run-s install-app-deps clean` especially because it must be `yarn run-s install-app-deps clean` according to the [package.json](https://github.com/RocketChat/Rocket.Chat.Electron/blob/3b2d0aaa51eaf96442c3084eaeb8621465d40cee/package.json#L23) and is the `postinstall` step. 

Furthermore running your used command but as mentioned in the package.json it succeeds:

```
$ yarn run-s install-app-deps clean
yarn run v1.22.10
$ /Users/dom/git/work/Rocket.Chat.Electron/node_modules/.bin/run-s install-app-deps clean
$ electron-builder install-app-deps
  • electron-builder  version=22.9.1
  • loaded configuration  file=/Users/dom/git/work/Rocket.Chat.Electron/electron-builder.json
$ rimraf app dist
✨  Done in 1.66s.
```

**Comment by jeanfbrito** [comment]
Please test if the latest release did solve this. If not please reopen the issue or create a new one and tell us. I will be closing this one for now.

Thank you!
